Documentation/developer/getting_started.rst

```diff
@@ -0,0 +1,223 @@
+
+Getting Started as a Contributer
+================================
+
+Icarus Verilog development is centered around the github repository at
+`github.com/steveicarus/iverilog <http://github.com/steveicarus/iverilog>`_.
+Contributing to Icarus Verilog requires a basic knowledge of git and github,
+so see the github documentation for more information. The sections below will
+step you through the basics of getting the source code from github, making a
+branch, and submitting a pull request for review.
+
+Getting Icarus Verilog
+----------------------
+
+To start, you will need to clone the code. It is preferred that you use the
+"ssh" method, and the ssh based clone with the command:
+
+.. code-block:: none
```

Comment:
```suggestion
.. code-block:: console
```

To indicate that this is an interactive shell session.

Reply:
Didn't know that was an option. Done.